components/org.wso2.carbon.identity.application.authenticator.fido2/src/main/java/org/wso2/carbon/identity/application/authenticator/fido2/core/WebAuthnService.java

```diff
@@ -825,8 +824,12 @@ private void addFIDO2Registration(PublicKeyCredentialCreationOptions publicKeyCr
 
         boolean requireResidentKey = false;
         if (publicKeyCredentialCreationOptions.getAuthenticatorSelection().isPresent()) {
```

Comment:
Can `publicKeyCredentialCreationOptions.getAuthenticatorSelection()` return null?

Reply:
It can not be null. https://github.com/Yubico/java-webauthn-server/blob/6b9937595efd4185fb166c78e6cdfc9497750ffd/webauthn-server-core/src/main/java/com/yubico/webauthn/data/PublicKeyCredentialCreationOptions.java#L227